provision.sh

```diff
@@ -40,7 +40,7 @@ KONG_ADMIN_LISTEN="0.0.0.0:8001"
 KONG_ADMIN_LISTEN_SSL="0.0.0.0:8444"
 
 if [ $KONG_NUM_VERSION -gt 001003 ]; then
-  KONG_DOWNLOAD_URL="https://bintray.com/kong/kong-deb/download_file?file_path=kong-${KONG_VERSION}.trusty.all.deb"
+  KONG_DOWNLOAD_URL="https://download.konghq.com/gateway-0.x-ubuntu-trusty/pool/all/k/kong-community-edition/kong-community-edition_${KONG_VERSION}_all.deb"
```

Comment:
As we have tags for historical versions, can we get rid of these `if`s handling old versions? cc @hutchic @Tieske

Reply:
well bintray is gone, so I see no reason to leave them in

Reply:
Bintray is gone, but old packages are also available in download.konghq.com, so that is not a blocker if there is a reason to keep those compatibility checks.

Reply:
created a card to check on simplifying this